build.xml

```diff
@@ -19,7 +19,7 @@
 
 	<property name="image-format" value="vmdk_only" />
 	<target name="ovf" depends="build,ovfOnly,ovfCopy" description="build all and generate vmidc ovf" />
-
+        <target name="dockerimage" depends="build,dockerImageBuild,clean" description="build all and generate docker image in tar" />
```

Comment:
why can't we separate out the build.xml for the container?

Reply:
@sudhirappaji  I followed legacy approach for building the image within osc-core.  Since osc-core can contain only one build.xml file, it may also confuse user if we have multiple build.xml files in same module. Please re-consider and let me know for further changes.

Reply:
combining both methods into one does not help... it is better to separate. lets get opinion from Arvind and Emanoel, how they look at this?

Reply:
given that ovf build xml's will be removed shortly, i would go with whatever is quickest to implement. To sudhir's poiint, Please keep an eye out so that we are able to strip out the ovf build portions easily without impacting the new docker portions use sections of the file etc